paper-badge.html

```diff
@@ -19,6 +20,14 @@
 the anchor element specified in the `for` attribute, or, if that doesn't exist,
 centered to the parent node containing it.
 
+Badges can also contain an icon by adding the `icon` attribute and setting
+it to the id of the desired icon. Please note that you should still set the
+`label` attribute in order to keep the element accessible.
+
+`paper-badge` includes the default `iron-icons` iconset, however you can use any icon you would
```

Comment:
Ah, I understand now. Do you think we should even ship the default set? I think: if you want to use an icon, you should import the set. What do you think?